proc/proc_linux.go

```diff
@@ -309,23 +311,38 @@ func (dbp *Process) trapWait(pid int) (*Thread, error) {
 	}
 }
 
-func status(pid int) rune {
+func (dbp *Process) status(pid int) rune {
```

Comment:
Instead of making these methods off `Process`, lets instead move the logic of parsing `comm` and storing it on `Process.comm` into `Process.LoadInformation`.